packages/@dcl/inspector/src/components/Controls/Controls.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react'
+import { MdOutlineDriveFileRenameOutline } from 'react-icons/md'
+import { AiFillFileAdd, AiFillDelete } from 'react-icons/ai'
+import { RxListBullet } from 'react-icons/rx'
+
+import './Controls.css'
+
+interface ControlsProps {
+  enableAdd?: boolean
+  enableEdit?: boolean
+  enableRemove?: boolean
+  handleEdit: (e: React.MouseEvent) => void
+  handleAdd: (e: React.MouseEvent) => void
+  handleRemove: (e: React.MouseEvent) => void
+}
+
+const cancelingKeys = new Set(['Escape', 'Tab'])
+
+const getActiveClass = (active: boolean) => active ? 'active' : ''
+
+const Controls = ({
+  enableAdd = true,
+  enableEdit = true,
+  enableRemove = true,
+  handleEdit,
+  handleAdd,
+  handleRemove
+}: ControlsProps) => {
+  const [active, setActive] = useState(false)
+
+  const onCancel = (e: Event) => {
+    e.stopPropagation()
+    setActive(false)
+  }
+
+  const onKeyUp = (e: KeyboardEvent) => {
+    if (cancelingKeys.has(e.key)) onCancel(e)
+  }
+
+  useEffect(() => {
+    if (active) {
+      document.body.addEventListener('click', onCancel)
+      document.body.addEventListener('keyup', onKeyUp)
+    } else {
+      document.body.removeEventListener('click', onCancel)
+      document.body.removeEventListener('keyup', onKeyUp)
+    }
+  }, [active])
+
+  const handleClick = (e: React.MouseEvent) => {
+    e.stopPropagation()
+    setActive(!active)
+  }
+
+  const handleAction = (cb: (e: React.MouseEvent) => void) => (e: React.MouseEvent) => {
+    setActive(false)
+    cb(e)
+  }
+
+  const someActionIsEnabled = enableAdd || enableEdit || enableRemove
+  const shouldRenderActions = someActionIsEnabled && active
+
+  return (
+    <div className={`component controls ${getActiveClass(active)}`}>
```

Comment:
Following the standards the outter class of the component should match the component name `Controls`.